packages/espresso/android/src/main/java/androidx/test/espresso/flutter/action/SyntheticClickAction.java

```diff
@@ -21,7 +21,7 @@
  * <p>Note, this is not a real click gesture event issued from Android system. Espresso delegates to
  * Flutter engine to perform the {@link SyntheticClick} action.
  */
-@Beta
+@ExperimentalTestApi
```

Comment:
Is it possible to move this annotation to methods instead of the whole class?

Reply:
Done